code/game/mecha/equipment/tools/mining_tools.dm

```diff
@@ -112,7 +112,7 @@
 	desc = "Equipment for engineering and combat exosuits. It will automatically check surrounding rock for useful minerals."
 	icon_state = "mecha_analyzer"
 	origin_tech = "materials=3;engineering=2"
-	equip_cooldown = 30
```

Comment:
Damn. I thought this only works for equipment that's actually useable, not the equipment that's constantly processed.

Anyway, nothing that bad happens. It just updates mineral overlays a lot more often. It would help if you will put up a fix, because my local repo is currently full of boards and frames changes.